cl/simple_pages/templates/contact_form.html

```diff
@@ -1,52 +1,53 @@
-{% extends "base.html" %}
+{% extends "base.html" %}{% load static text_filters %}
 
 {% block title %}Contact Us – CourtListener.com{% endblock %}
 
 {% block sidebar %}{% endblock %}
 
+{% block head %}
+  {# Fetch issue types and terms URL #}
+  {{ form.TECH_ISSUE_TYPES|uniq|json_script:"tech-types" }}
+  <script id="terms-url">{% url "terms" %}#removal</script>
```

Comment:
I'm getting this console error:
```Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' 'report-sample' https://dev-com-courtlistener-storage.s3.amazonaws.com/ https://hcaptcha.com/ https://*.hcaptcha.com/ https://plausible.io/ 'nonce-H1gWzELYgymoa4fFflodtw=='". Either the 'unsafe-inline' keyword, a hash ('sha256-JYNdMEctaDVeJ2u6yMc8m0NmyMStb6bF4C7zfR2T060='), or a nonce ('nonce-...') is required to enable inline execution.```

I tried this and the error went away. I’m not sure if it’s the best practice.

` <script id="terms-url" type="application/json">{% url "terms" %}#removal</script>`

Or if it would be better to add `nonce="{{ request.csp_nonce }}"`